src/main/java/org/intellij/sdk/codesync/database/Database.java

```diff
@@ -21,6 +21,12 @@ public static boolean isConnected(){
         return connection != null;
     }
 
+    public static void checkConnection() throws SQLException {
```

Comment:
Thanks @syedgulahmed. Can we add a test for this? i.e. explicitlty close a connection and the try to run the query?